pysrc/models/metanome_run.py

```diff
@@ -110,6 +126,28 @@ def parse_results(result_file_name: str, arity: str, results_folder: str, print_
 
             # ind = f'{" & ".join(dependant_list)} [= {" & ".join(referenced_list)}'
             ind = IND(dependents=dependant_list, referenced=referenced_list)
+
+        elif arity == 'nary' and is_baseline == False:
+            dependant_list: list[ColumnInformation] = []
+            dependant_raw = line_json['dependant']['columnIdentifiers']
+            for dependant_entry in dependant_raw:
+                dependant_table = dependant_entry['tableIdentifier'].rsplit('.', 1)[0].rsplit('_', 1)[0]
+                dependant_column = 'column' + str(dependant_entry['tableIdentifier'].rsplit('.', 1)[0].rsplit('_')[-1])
+                dependant = ColumnInformation(table_name=dependant_table, column_name=dependant_column)
+                # dependant_list.append(f'{dependant_table}.{dependant_column}')
+                dependant_list.append(dependant)
+
+            referenced_list: list[ColumnInformation] = []
```

Comment:
```suggestion
            dependant_list = []
            dependant_raw = line_json['dependant']['columnIdentifiers']
            for dependant_entry in dependant_raw:
                dependant_table = dependant_entry['tableIdentifier'].rsplit('.', 1)[0].rsplit('_', 1)[0]
                dependant_column = 'column' + str(dependant_entry['tableIdentifier'].rsplit('.', 1)[0].rsplit('_')[-1])
                dependant = ColumnInformation(table_name=dependant_table, column_name=dependant_column)
                # dependant_list.append(f'{dependant_table}.{dependant_column}')
                dependant_list.append(dependant)

            referenced_list = []
```

Python takes the types from above.